### PR TITLE
add readme with screenshots

### PR DESCRIPTION
**Pull request by trepmal**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[![Run PHPUnit and PHPCS](https://github.com/Automattic/maintenance-mode-wp/actions/workflows/integrate.yml/badge.svg)](https://github.com/Automattic/maintenance-mode-wp/actions/workflows/integrate.yml)
+
+# Maintenance Mode
+
+Shut down your site for a little while and do some maintenance on it!
+
+Usage:
+
+ - Add a template to your theme's root folder called `template-maintenance-mode.php`.
```
Comment (mikeyarce) [inline readme.md:9]: Is this required or optional?

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ - This should be a simple HTML page that should include the message you want to show your visitors.
+ - Note: the template should include `wp_head()` and `wp_footer()` calls.
+ - Add the VIP_MAINTENANCE_MODE constant to your theme and set to `true`.
```
Comment (mikeyarce) [inline readme.md]: Should we add back-ticks around  `VIP_MAINTENANCE_MODE`?

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+## Installation
+
+1. Install the plugin.
+1. Set the `VIP_MAINTENANCE_MODE` constant to true.
+1. Activate.
+
+## Default template screenshots
+
+twentytwentyone:  
+![twentytwentyone](screenshot-twentytwentyone.png)
```
Comment (mikeyarce) [inline readme.md]: Accessibility-wise, should we change the Alt description to something like

`Screenshot of Maintenance Mode active on the Twenty Twenty One theme`

**Review by mikeyarce — APPROVED** [review]
I know this is copy-pasted from the `txt` file, but I did add some comments for things that we might want to improve.

File: readme.md (hunk 1 above)
Comment (trepmal) [reply readme.md:9]: I think this should be spun off into a separate issue, then update both readmes with any instructional improvements

**Pull request merged**